src/components/ChecAlert.vue

```diff
@@ -111,6 +112,10 @@ $alert-inline-colors: (
     'border': 'orange-400',
   ),
   'info': (
+    'background': 'blue',
+    'border': 'blue-500',
```

Comment:
The other colour variants (excluding gray) all use 400, do you want to keep 500 here?

Reply:
Nice spotting. 🙈

Reply:
Yea it makes it stand out a bit more and matches with the other variants a bit better.